### PR TITLE
Remove coverall suport, install codecov in travis_after_test_success

### LINKED ISSUE #642: coverage reporting broken?
See for example: https://travis-ci.com/OCA/brand/jobs/293455206#L1189-L1199

I have not had time to investigate yet.

### PR DESCRIPTION
**Pull request by sbidoul**
As recommended by https://docs.coveralls.io/python

fixes #642

### CODE REVIEW

**Comment by pedrobaeza** [comment]
We should remove all the coveralls remnants instead. We are using codecov since a lot.

**Comment by sbidoul** [comment]
still WIP

File: .travis.yml [hunk 1]
```diff
@@ -69,12 +69,12 @@ install:
...
   - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
    # Force install lint requirements in all cases of MQT for self_test script.
-  - pip install codecov==2.0.15 coveralls==1.8.2
+  - pip install coverage
   - LINT_CHECK="1" travis_install_nightly 8.0 # only used if VERSION not set in env
   - git --git-dir=${TRAVIS_BUILD_DIR}/.git add --all  # All modules moved are modules changed to test PR changes
 
 script:
-  - TRAVIS_PULL_REQUEST="1" coverage run --append ./travis/travis_run_tests 8.0  # only used if VERSION not set in env
+  - TRAVIS_PULL_REQUEST="1" coverage run ./travis/travis_run_tests 8.0  # only used if VERSION not set in env
```
Comment (sbidoul) [inline .travis.yml]: I think the root cause was this.

**Comment by sbidoul** [comment]
Ok, this [works](https://travis-ci.org/github/OCA/mis-builder/jobs/665236888?utm_medium=notification&utm_source=github_status).

Good to go.

**Comment by moylop260** [comment]
Thanks for fixing @sbidoul 💯

**Pull request merged**